content/forking-a-core-package/sections/creating-a-fork-of-a-core-package-in-atom-atom.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Creating a Fork of a Core Package in atom/atom
+---
+### Creating a Fork of a Core Package in atom/atom
+
+Several of Atom's core packages are maintained in the [`packages` directory of the atom/atom repository](https://github.com/atom/atom/tree/master/packages). If you would like to use one of these packages as a starting point for your own package, please follow the steps below.
+
+#### Creating Your New Package
+
+For the sake of this guide, let's assume that you want to start with the current code in the [one-light-ui](https://github.com/atom/atom/tree/master/packages/one-light-ui) package, make some customizations to it, and publish your new package under the name "one-light-ui-plus".
+
+1. Download the [current contents of the atom/atom repository as a gzipped tarball](https://github.com/atom/atom/archive/master.tar.gz)
+
+2. Unzip the file to a temporary location (for example `/tmp/atom`)
```

Comment:
You can have separate example paths for Windows and Linux/Mac by using [tagged spans](https://github.com/atom/flight-manual.atom.io/blob/master/CONTRIBUTING.md#os-specific-sections):

```markdown
(for example <span class='platform-mac platform-linux'>`/tmp/atom`</span><span class='platform-windows'>`C:\TEMP\atom`</span>)
```